notices.json

```diff
@@ -68,18 +68,18 @@
     }
   },
   {
-    "id": "server_upgrade_v10.11",
+    "id": "server_upgrade_v10.12",
     "conditions": {
       "audience": "sysadmin",
       "clientType": "all",
-      "serverVersion": ["<10.11"],
+      "serverVersion": ["<10.12"],
       "instanceType": "onprem",
-      "displayDate": ">= 2025-08-16T00:00:00Z"
+      "displayDate": ">= 2025-09-16T00:00:00Z"
     },
     "localizedMessages": {
       "en": {
-        "title": "Mattermost 10.11 is here!",
-        "description": "Mattermost v10.11 Extended Support Release contains multiple new quality of life improvements, including server support for [Playbooks on mobile]​(https://docs.mattermost.com/end-user-guide/workflow-automation.html#access). [Upgrading](https://docs.mattermost.com/upgrade/upgrading-mattermost-server.html) only takes a few minutes.",
+        "title": "Mattermost 10.12 is here!",
+        "description": "Mattermost v10.12 Extended Support Release contains multiple new quality of life improvements, including server support for [Playbooks on mobile]​(https://docs.mattermost.com/end-user-guide/workflow-automation.html#access). [Upgrading](https://docs.mattermost.com/upgrade/upgrading-mattermost-server.html) only takes a few minutes.",
```

Comment:
The description here may need to be updated based on https://github.com/mattermost/notices/pull/443.